.travis.yml

```diff
@@ -11,6 +12,11 @@ install:
 script:
   - xbuild /p:Configuration=Release ./OptimizelySDK.Travis.sln
   - mono ./testrunner/NUnit.Runners.2.6.4/tools/nunit-console.exe ./OptimizelySDK.Tests/bin/Release/OptimizelySDK.Tests.dll 
+after_success:
+  - pip install --user awscli
+  - (AWS_ACCESS_KEY_ID=$OFTA_KEY AWS_SECRET_ACCESS_KEY=$OFTA_SECRET AWS_DEFAULT_REGION=$OFTA_REGION aws s3 cp ./OptimizelySDK/bin/Release/OptimizelySDK.dll s3://optly-fs-travisci-artifacts/${TRAVIS_REPO_SLUG}/${TRAVIS_BUILD_NUMBER}/${TRAVIS_JOB_NUMBER}/OptimizelySDK.dll-unsigned)
```

Comment:
Can we consolidate above and last 2 lines to single line something like. 
`find ./ -type f | grep pattern | xargs ...`
All dlls will follow the same pattern and also name must follow OptimizelySDK.*.dll

Reply:
fixed with a combination of find and while loop.